src/worker.cc

```diff
@@ -86,23 +89,49 @@ void Worker::newConnection(evconnlistener *listener, evutil_socket_t fd,
                     Redis::Connection::OnEvent, conn);
   bufferevent_enable(bev, EV_READ);
   Status status = worker->AddConnection(conn);
+  if (!status.IsOK()) {
+    std::string err_msg = Redis::Error("ERR " + status.Msg());
+    write(fd, err_msg.data(), err_msg.size());
+    conn->Close();
```

Comment:
Need to return here after closing the connection

Reply:
Should I do it in a separate commit or I can change this commit + make a force push to branch?

Reply:
It's fine to add a new commit to fix it. 👍

Reply:
Done. And in line 128 too.